### ISSUE TITLE
As a demo app, can we remove the password policy?

### ISSUE THREAD

**Issue by leedstyh** [AnomalyInnovations/serverless-stack-com#216]
I mean this [demo app](https://demo.serverless-stack.com/signup).

Tried more than 5 times, still get `Password did not confirm with policy: XXX`

Or show the full policy on the sign up page!

**Comment by jayair** [comment]
@leedstyh Yeah it is a bit annoying. We don't touch the default AWS Cognito policy. You can see it here  - https://d33wubrfki0l68.cloudfront.net/1f82c4f88252f824b6897ff7f93ff76e2e3912d6/b9440/assets/cognito-user-pool/select-create-pool.png

But as you working on the tutorial you can change the policy. It's a part of this chapter - https://serverless-stack.com/chapters/create-a-cognito-user-pool.html
